lib/util/index.js

```diff
@@ -588,11 +588,23 @@ function createRoomConnectEventPayload(connectOptions) {
     payload[eventProp] = boolToString(!!connectOptions[prop]);
   });
 
-  // numbers and string properties.
-  [['maxVideoBitrate'], ['maxAudioBitrate'], ['iceTransportPolicy'], ['region'], ['name', 'roomName']].forEach(([prop, eventProp]) => {
+  // numbers properties.
+  [['maxVideoBitrate'], ['maxAudioBitrate']].forEach(([prop, eventProp]) => {
     eventProp = eventProp || prop;
-    if (typeof connectOptions[prop] === 'number' || typeof connectOptions[prop] === 'string') {
+    if (typeof connectOptions[prop] === 'number') {
       payload[eventProp] = connectOptions[prop];
+    } else if (typeof connectOptions[prop] === 'string') {
+      payload[eventProp] = Number(connectOptions[prop]);
+    }
```

Comment:
I think we need to ensure that we do not send 'NaN' values.
```suggestion
    } else if (typeof Number(connectOptions[prop]) === 'number') {
      payload[eventProp] = Number(connectOptions[prop]);
    }
```